cloudfoundry-operations/src/main/lombok/org/cloudfoundry/operations/buildpacks/BuildpackSummary.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.operations.buildpacks;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * A Cloud Foundry Buildpack Summary
+ */
+@Data
+public final class BuildpackSummary {
```

Comment:
@antechrestos We've gone back and forth on this quite a bit, but our current behavior is that _at the operations layer_ we're going to use the simplest possible names (`Buildpack` instead of `BuildpackSummary`) **unless** two related operations return a `*Detail` / `*Summary` pair.

I'll make this change, and don't feel bad that you didn't get it right.  I've had the team working on a final decision and it's likely that our current code is inconsistent.